util.py

```diff
@@ -4,32 +4,63 @@
 
 from constants import *
 
+def get_optimizer_enum_from_string(optimizer_str):
+    try:
+        optimizer = OptimizerType[optimizer_str]
+    except KeyError:
+        print('OptimizerType was not found in possible options.')
+        print('Available options are:')
+        available_optimizers = [name for name, member in OptimizerType.__members__.items()]
+        print(available_optimizers)
+        exit(1)
+    return optimizer
+
+def validate_activation(activation):
+    try:
+        activ = OutputActivation[activation]
+    except KeyError:
+        print('OutputActivation was not found in possible options.')
+        print('Available options are:')
+        available_activs = [name for name, member in OutputActivation.__members__.items()]
+        print(available_activs)
+        exit(1)
+    return activ
 
 def find_resume_json_weights_args(args):
-    return find_resume_json_weights_str(args.model_save, args.resume_subdir, args.resume_epoch)
+    args.resume_json, args.resume_weights, args.resume_epoch = find_resume_json_weights_str(args.model_save, args.resume_subdir, args.resume_epoch)
```

Comment:
Just double checking, does this no longer need to be returned?

Reply:
Yeah it was a bug. Shouldn't have been returned in the first place